src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createTextNode = (type, parentNode, classes, text) => {
+  const node = document.createElement(type);
+
+  node.classList.add(...classes);
+  node.textContent = text;
+  parentNode.append(node);
+
+  return node;
+};
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
```

Comment:
The `promise2` is rejected after 3 seconds. This might confuse users if they don't see an error message immediately. Consider providing more feedback to the user about what's happening in the background.